schema/defs.yaml

```diff
@@ -313,3 +314,21 @@ description: Common schema definitions shared by all themes
         moving along the line from beginning to end
       properties:
         side: { "$ref": "#/$defs/propertyDefinitions/side" }
+    cartographyContainer:
+      title: cartography
+      description: defines cartograhic hints for optimal use of Overture features in map making
+      type: object
+      properties:
+        cartography:
+          type: object
+          properties:
+            min_zoom:
+              description: recommended minimum tile zoom per the google tiling specification
+              type: integer
+              minimum: 0
+              maximum: 23
+            max_zoom:
+              description: recommended maximum tile zoom per the google tiling specification
```

Comment:
Can we do sentence case, capitalize "G" for Google, and link a URL if possible?

Just curiosity: Why did you go with Google rather than Slippy Maps/Bing?

Reply:
Bing is off by one (starts at 1 instead of 0), the vector tile spec starts at 0 as defined by Google